doc/developers/develop.rst

```diff
@@ -532,6 +532,9 @@ general only be determined at runtime.
 
 The current set of estimator tags are:
 
+Array_API_support (default=False)
```

Comment:
all other tags are all small case, i.e. this would be `array_api_support`, and should be after `allow_nan`

Reply:
How is the sorting determined?